### PR TITLE
PoS: active validator limit

### PR DESCRIPTION
**Pull request by hsutaiyu**
- Update active validator limit
- Language fixes
- Remove/update outdated info

### CODE REVIEW

File: docs/pos/get-started/becoming-a-validator.md [hunk 1]
```diff
@@ -8,12 +8,13 @@ Validators play a crucial role in maintaining the integrity and security of the
...
+!!! info "Active validator limit"
+    
+    There is limited space for accepting new validators. Currently, the system allows for a maximum of 105 active validators at any given time. New validators can join the active set only when a currently active validator unbonds or is removed due to low performance.
     
-    There is limited space for accepting new validators. New validators can only join the active set when a currently active validator unbonds/is removed due to low performance. 
-    If you are interested in becoming a validator on the PoS network, you can submit an application here: https://polygoncommunity.typeform.com/validatorshub
+    If you are interested in becoming a validator on the PoS network, you can submit an application over at the [Polygon validators hub](https://polygoncommunity.typeform.com/validatorshub).
 
-    * Submitting an application does not guarantee you a validator slot.
+    \* *Submitting an application does not guarantee you a validator slot.*
```
Comment (hsutaiyu) [inline docs/pos/get-started/becoming-a-validator.md]:
```suggestion
    \* *Submitting an application does not guarantee a validator slot.*
```

File: docs/pos/get-started/becoming-a-validator.md [hunk 2]
```diff
@@ -69,11 +70,9 @@ Bor is the block producing node and layer for the Polygon PoS network. It is bas
 
 A blockchain validator is someone who is responsible for validating transactions within a blockchain. On the Polygon PoS network, any participant can be qualified to become a Polygon's validator by running a validator node (sentry + validator) to earn rewards and collect transaction fees. To ensure the good participation by validators, they lock up at least 1 MATIC token as a stake in the ecosystem.
 
-!!! info "Active validator limit"
-    
-    Currently, there is a limit of 100 active validators at a time.
+!!! info "PIP4 raises the minimum staking amount"
 
-    Also, after the [<ins>PIP4 governance proposal</ins>](https://forum.polygon.technology/t/pip-4-validator-performance-management/9956) is implemented on the contract-level, the minimum staking amount will increase to 10,000 MATIC.
+    After the implementation of the [PIP4 governance proposal](https://forum.polygon.technology/t/pip-4-validator-performance-management/9956) at the contract level, the minimum staking amount will increase to *10,000 MATIC*.
```
Comment (kmurphypolygon) [inline docs/pos/get-started/becoming-a-validator.md:75]:
```suggestion
    After the implementation of the [PIP4 governance proposal](https://forum.polygon.technology/t/pip-4-validator-performance-management/9956), the minimum staking amount will increase to *10,000 MATIC* at the contract level.
```

Comment (kmurphypolygon) [inline docs/pos/get-started/becoming-a-validator.md:75]:
```suggestion
    After the implementation of the [PIP4 governance proposal](https://forum.polygon.technology/t/pip-4-validator-performance-management/9956), the minimum staking amount will increase to *10,000 MATIC*.
```

File: docs/pos/reference/contracts/stakingmanager.md [hunk 3]
```diff
@@ -13,16 +13,16 @@ For the Polygon PoS Proof of Security based consensus, all the $2/3+1$ proof ver
 
 ### Admissions
 
-At present, there are no open validator slots available on Polygon PoS. There is also a wait list to become a validator. In the future, if slots become available, validators may apply to be considered and removed off of the wait list.
+Currently, the system allows for a maximum of 105 active validators at any given time on Polygon PoS. There is also a waitlist to become a validator. New validators can join the active set only when a currently active validator unbonds or is removed due to low performance. You can apply to become a validator via the [Polygon validators hub](https://polygoncommunity.typeform.com/validatorshub).
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/stakingmanager.md:16]: duplicate content ?

**Review by kmurphypolygon — APPROVED** [review]
LGTM

File: docs/pos/reference/contracts/stakingmanager.md (hunk 3 above)
Comment (hsutaiyu) [reply docs/pos/reference/contracts/stakingmanager.md:16]: Different page.

Comment (kmurphypolygon) [reply docs/pos/reference/contracts/stakingmanager.md:16]: yeah, i know ...

**Pull request merged**